string params #json parameters
---
bool status
